protocol-impl/src/main/java/io/camunda/zeebe/protocol/impl/stream/job/JobActivationProperties.java

```diff
@@ -41,4 +42,15 @@ public interface JobActivationProperties extends BufferReader, BufferWriter {
    * @see JobRecordValue#getDeadline()
    */
   long timeout();
+
+  /**
+   * When jobs are streamed from the engine, it is possible that the jobs belong to different
+   * tenants.
+   *
+   * <p>This can be useful when requesting jobs for multiple tenants at once. Each of the activated
+   * jobs will be owned by the tenant that owns the corresponding process instance.
```

Comment:
🔧 Instead of focusing on how jobs streamed from the engine could belong to different tenants, I suggest we flip this around to activating as an action, instead of something that the engine does for you. This fits the context of this class better, which is about the properties of the jobs to activate for a stream that is being opened.

> A stream can be opened to activate the jobs of multiple tenants. Each activated job will be owned by the tenant that owns the corresponding process instance.

Reply:
@korthout I rephrased the javadoc. To make faster progress, I'll merge this PR. In case you have more feedback, I can implement the changes in later multi-tenancy-related PRs.

Reply:
💭 @koevskinikola Please feel free to merge after I've approved a pull request and when you make some changes due to suggestions unless you feel those changes need a second pair of eyes. In cases where I feel I want to have another look, I'll typically reject the changes to force another review.